DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -193,8 +193,8 @@
     <![CDATA[superClusterRelinked_.reset();]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="14">
-   <version ClassVersion="15" checksum="1826981639"/>
+  <class name="pat::Jet"  ClassVersion="15">
+   <version ClassVersion="15" checksum="727883729"/>
```

Comment:
@slava77 Thank you for spotting this!
This is a conflict in version numbers. However, this problem predates this PR as 7_4_X and 7_4_ROOT6_X already conflict.  So, no matter what is done in 7_5_X, there will still be a conflict. So, this conflict cannot be solved correctly simply by merging things differently.  This file needs fixes in all three releases.
The simplest way out of this may be to go ahead with this merge, and I'll issue PR's for this file after the merge in all three releases fixing things.
@ktf If you have any other ideas pleas let me know.